src/App.js

```diff
@@ -1,29 +1,34 @@
 import React from 'react'
+import styled from 'styled-components/macro'
 import PodcastListPage from './pages/PodcastListPage/PodcastListPage'
 import { Route, Switch, useHistory } from 'react-router-dom'
 import DetailsPage from './pages/DetailsPage/DetailsPage'
 import FavoritesPage from './pages/FavoritesPage/FavoritesPage'
+import Navigation from './components/Navigation/Navigation'
 
-function App() {
+export default function App() {
   const { push } = useHistory()
 
   return (
-    <Switch>
-      <Route exact path="/">
-        <PodcastListPage onClickFavorites={handleClickFavoritesPage} />
-      </Route>
-      <Route path="/favorites">
-        <FavoritesPage />
-      </Route>
-      <Route path="/details">
-        <DetailsPage />
-      </Route>
-    </Switch>
+    <AppContainer>
+      <Switch>
+        <Route exact path="/">
+          <PodcastListPage />
+        </Route>
+        <Route path="/favorites">
+          <FavoritesPage />
+        </Route>
+        <Route path="/details">
+          <DetailsPage />
+        </Route>
+      </Switch>
+      <Navigation />
+    </AppContainer>
   )
-
-  function handleClickFavoritesPage() {
-    push('/favorites')
-  }
 }
 
-export default App
+const AppContainer = styled.section`
+  height: 100vh;
+  display: grid;
+  grid-template-rows: 1fr 48px;
```

Comment:
Maybe use relative size 48px = 3em :)

Reply:
Thank you for your review. Updated accordingly.